src/components/firebase-access.jsx

```diff
@@ -396,50 +404,3 @@ export const getUserData = async (roomID, userID) => {
 export const getUserRef = (roomID, userID) => {
     return db.collection('Rooms').doc(roomID).collection('users').doc(userID);
 }
-
```

Comment:
Dont delete the methods leave them there just put it as null for rn.

Reply:
i think they're easily recoverable since they're all on github, so it's probably ok if we just delete them for now since they're unused. we can add them back in if we have time to add the config page back in.

Reply:
I was going to add them back in today so I'd rather them not be deleted comment them out if you want , but don't just delete them.

Reply:
added back in, commented out.